ledger-service/http-json/src/main/scala/com/digitalasset/http/WebSocketService.scala

```diff
@@ -257,11 +258,17 @@ object WebSocketService {
             .toLeftDisjunction(x.ekey.templateId)
         }
 
-      val q: Map[domain.TemplateId.RequiredPkg, LfV] = resolvedWithKey.toMap
+      val q: Map[domain.TemplateId.RequiredPkg, HashSet[LfV]] =
+        resolvedWithKey.foldLeft(Map[domain.TemplateId.RequiredPkg, HashSet[LfV]]())((acc, el) =>
```

Comment:
```suggestion
        resolvedWithKey.foldLeft(Map.empty[domain.TemplateId.RequiredPkg, HashSet[LfV]])((acc, el) =>
```
Same bikeshedding as on the other PR. I find that a bit easier to read.

Reply:
Done.